tests/create_guardduty_member/main.tf

```diff
@@ -1,37 +1,42 @@
+# The provider account for the GuardDuty member account
 provider "aws" {
   region  = "us-east-1"
-  profile = "resource-member"
+  profile = "plus3it-member" # Profile must exist in your .aws/config
 }
 
+# AWS provider account for the GuardDuty primary account
 provider "aws" {
   region  = "us-east-1"
-  alias   = "resource-owner"
-  profile = "resource-owner"
+  alias   = "administrator"
+  profile = "plus3it-management" # Profile must exist in your .aws/config
 }
 
+# Create GuardDuty detector for the administrator account
+resource "aws_guardduty_detector" "administrator" {
+  provider = aws.administrator
+  enable   = true
+}
+
+# Invite this member account to join the administrator account GuardDuty organization.
+# - Creates a GuardDuty detector for the member account
+# - Creates a GuardDuty member resource in the administrator account which imnvites the member account to join the administrator account GuardDuty organization.
+# - Creates a GuardDuty invite accepter in the member account to accept the invite from the administrator account
 module "guardduty_member" {
-  source = "../../"
+  source = "../../modules/guardduty-member-account"
+
+  enable = true
 
   providers = {
-    aws        = aws
-    aws.master = aws.resource-owner
+    aws               = aws
+    aws.administrator = aws.administrator
   }
 
-  guardduty_master_detector_id = aws_guardduty_detector.master.id
-  email_address                = var.member_email
-
-  depends_on = [
-    aws_guardduty_detector.master
-  ]
-}
-
-resource "aws_guardduty_detector" "master" {
-  provider = aws.resource-owner
-
-  enable = true
-}
+  member = {
+    email                      = "aws-accounts+tardigrade-dev-tenant-001@plus3it.com"
```

Comment:
This needs to remain a variable.

Reply:
It is. Please see modules/member/variables.tf.

Reply:
No, I mean it needs to remain a variable in the tests. Imagine a non-p3 contributor who does not have access to our test accounts....

Reply:
Oh...so you want the user to have to set this from the command line?

Reply:
Updated so the user must enter the email address from the command line.